acpi.adoc

```diff
@@ -15,8 +15,7 @@ A compliant system must meet the following requirements:
 
 === ACPI-specific Hardware Requirements
 
-This section lists the BRS-specific hardware requirements imposed by
-ACPI support.
+This section lists the requirements imposed by ACPI support.
```

Comment:
Darius, could you pull out this commit into a separate PR so we can merge this one?

Reply:
#38

Reply:
Thanks. Applied.